### ISSUE TITLE
[OTHER] Running the repo

### ISSUE THREAD

**Issue by MartinDM** [wet-boew/wet-boew#9086]
The Readme file does not indicate the usage of the npm scripts defined in `package.json`.
Are any of those used for running the project itself (it seems they aren't for that purpose?)

The Documentation is unclear on how to start the project, and refers to Documentation with the CG theme, which I don't see available in this repo. The [CG Theme repo](https://github.com/wet-boew/GCWeb/) also does not indicate how to run the application.

However, I have run the `setup` npm command I found in `script.md` , but am getting an error related to the Grunt version in a peer dependency.

![image](https://user-images.githubusercontent.com/7467069/115584274-927db880-a2c2-11eb-97e8-ad980ebc4bfe.png)

Is this something that has been addressed for this repo?

I have:
- ensured bower and grunt-cli is removed globally
- Ran `run npm i` - Causes peer dependency error above
- Ran `npm i  --legacy-peer-deps` - Also results in same error

Is this repo expected to build, please? 
Has any work been done to correct/allow the use of the specified version `^1.3.0`? as this project won't build for me at the moment. Any assistance or indication whether the repo is maintained would be a great help! :)

**Comment by thekodester** [comment]
I had no issues creating a fork and running the setup script:
```
git clone https://github.com/[username]/GCWeb.git
cd GCWeb
./script/setup 
grunt dist
```
NPM is not really used much, just some packages (mainly grunt) for building the distribution files (creates a folder called dist with the files).

**Comment by MartinDM** [comment]
OK, so the npm commands themselves aren't used for building the project and generating the output? Looks like this is all the `script` is doing anyway!
I hadn't run the last command in your example `grunt dist` - is this documented anywhere when getting started at all? (May have missed it)

However, when running that line, I'm getting the following :(

![image](https://user-images.githubusercontent.com/7467069/115681390-6575e800-a34c-11eb-8225-968f46326784.png)

Seems a bit of a generic error and doesn't seem to point to anything in particular...

**Comment by thekodester** [comment]
Yeah just looks like the normal warnings as wet uses some out of date npm packages for creating it distribution files.

**Comment by thekodester** [comment]
This page goes over it: https://wet-boew.github.io/wet-boew/docs/start-en.html

But their is a lot on that page that is out of date.

**Comment by thekodester** [comment]
So currently wet-boew and gcweb are the only ones maintained. You can read more here: https://github.com/wet-boew/wet-boew/issues/8366
